resolwe/flow/executors/docker/run.py

```diff
@@ -370,6 +377,7 @@ async def start(self):
         # container due to name clash.
         init_container_name = f"{self.container_name}-{_random_string()}-init"
 
+        # Init container must run as root to use named volumes.
```

Comment:
Move this comment just above the `"user": "0:0"` line.

Reply:
Done.